lib/pure/cookies.nim

```diff
@@ -50,6 +50,9 @@ proc setCookie*(key, value: string, domain = "", path = "",
                 maxAge = none(int), sameSite = SameSite.Default): string =
   ## Creates a command in the format of
   ## `Set-Cookie: key=value; Domain=...; ...`
+  ##
+
+  ## .. tip: Cookies can be vulnerable. Consider setting `secure=true`, `httpOnly=true` and `sameSite=Strict`.
```

Comment:
This is not correct and doesn't generate docs.
`Warning: RST style: double colon :: may be missing at end of 'tip' [warnRstStyle]`

Reply:
So it is better to use warnAsErrors when building our docs for CI.